### PR TITLE
Publishing corpus paralelo lfaligner

### PR DESCRIPTION
**Pull request by mariajoafana**
This will be soon our first original Spanish-language lesson :)

### CODE REVIEW

**Comment by mariajoafana** [comment]
I'm waiting for the author's bio

**Comment by vgayolrs** [comment]
Hola @mariajoafana Hay que incluir tópicos de la lección y la bio del autor para que Travis pueda pasar la confirmación.

**Comment by vgayolrs** [comment]
El tópico tiene que estar en inglés: "distant-reading"

**Review by vgayolrs — CHANGES_REQUESTED** [review]
Cambiar en "topics" "lectura distante" por "distant-reading"

**Comment by arojascastro** [comment]
Cool! Genial... por otra parte, ¿es realmente "lectura distante"? No sé, quizás necesitamos otro tópico para esta lección.

**Comment by mariajoafana** [comment]
> Cool! Genial... por otra parte, ¿es realmente "lectura distante"? No sé, quizás necesitamos otro tópico para esta lección.

¿Será que "parallel corpus" o "paralell alignment" funcionan como tópicos?

**Comment by rivaquiroga** [comment]
Quizás podría ser un tópico más general, como corpus "corpus building".

**Comment by vgayolrs** [comment]
@mariajoafana @rivaquiroga @arojascastro : los tópicos están pre-definidos:
Captura de Pantalla 2019-03-27 a la(s) 10 20 30
Si creen que hay que incluir otro, habría que abrir un ticket, discutirlo con los demás equipos y pedir a Matt que lo incluya para todos las versiones de PH en el archivo respectivo.

**Comment by arojascastro** [comment]
Cierto, pero se pueden crear más, supongo. Pero, bueno, era solo una sugerencia, tampoco me parece super importante.

**Comment by vgayolrs** [comment]
[aquí](https://github.com/programminghistorian/jekyll/blob/gh-pages/_data/topics.yml) está la lista completa con traducción de los tópicos. Esa es la que habría que modificar para incluir el que se considere.

**Comment by mariajoafana** [comment]
Ok, por el momento no propongamos otro tópico para este tutorial para que lo podamos publicar pronto, pero ya queda claro, gracias @vgayolrs, qué tenemos que hacer para proponer uno nuevo

**Review by vgayolrs — DISMISSED** [review]

**Review by arojascastro — DISMISSED** [review]

**Comment by arojascastro** [comment]
I don't know why this is not passing Travis. We get the following message:

1.44s$ _build/build.sh
Configuration file: /home/travis/build/programminghistorian/jekyll/_config.yml
            Source: /home/travis/build/programminghistorian/jekyll
       Destination: /home/travis/build/programminghistorian/jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
jekyll 3.7.4 | Error:  undefined method `each' for "distant-reading":String
The command "_build/build.sh" exited with 1.

I checked the YAML block and `distant-reading` is well written. Any idea @mdlincoln what's going on? Thank you.

**Comment by mdlincoln** [comment]
Oh, I see - it's a tiny error but:

```
topics: distant-reading
```

in yaml is a one-to-one piece of data. `topics` is like `authors`, though - it can have one or _more_ values. So you need to write it like a list

```
topics: 
    - distant-reading
```

(when jekyll complains `Error: undefined method 'each' for "distant-reading":String` that's because it tried to treat `"distant-reading"` like a list and got confused when it didn't see a list. That error message is... technically correct... but it doesn't really explain the root of the problem)

**Comment by arojascastro** [comment]
Thank you @mdlincoln ! 

Now we got this error

* In /es/lecciones/corpus-paralelo-lfaligner.md:
'Armando Luza' is not currently listed in ph_authors.yml. Check your spelling.
jekyll 3.7.4 | Error:  There were YAML errors.
The command "_build/build.sh" exited with 1.

I'll check that.

**Comment by mdlincoln** [comment]
check the spelling in `ph_authors.yml` 😆 it's slightly off

**Comment by arojascastro** [comment]
Yup! Done 🥇

**Comment by arojascastro** [comment]
Never ending:

 ./_site/es/lecciones/corpus-paralelo-lfaligner.html
  *  internally linking to "/"/es/lecciones/analisis-de-corpus-con-antconc, which does not exist (line 870)
     <a href="%22/%22/es/lecciones/analisis-de-corpus-con-antconc">AntConc</a>
htmlproofer 3.10.2 | Error:  HTML-Proofer found 1 failure!
The command "_build/build.sh" exited with 1.

**Comment by arojascastro** [comment]
Yay 🎆

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+Es importante ser sistemático con la clasificación de los documentos. El nombre de nuestros archivos txt debe acompañarse con el código que alude a la lengua del texto. Con ello aseguramos que la información con la que trabajamos siga convenciones oficiales que serán útiles a la hora de comunicar los resultados de nuestra investigación Para ello nos basaremos en el código [ISO 639-1](http://utils.mucattu.com/iso_639-1.html) que identifica a cada lengua con dos letras. Así, el español se identifica con *es*, el inglés con *en*, el francés con *fr* y el portugués con *pt*.
+
+Si trabajas con lenguas que no estén incluidas en ese código, puedes recurrir al código [ISO 639-3](https://www.ecured.cu/ISO_639-3) que utiliza descriptores de 3 letras y abarca la totalidad de las lenguas del mundo.
+
+Como resultado final del procesamiento de los textos con LF Aligner obtendrás algo así:
+
+{% include figure.html filename="lfaligner-1.jpg" %}
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: This needs a caption

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Además, si trabajas en un entorno Linux de 64 bits, necesitarás instalar algunos paquetes adicionales para que el programa funcione correctamente. Los comandos que debes introducir en la terminal son los siguientes:
+
+1. sudo dpkg --add-architecture i386
+2. sudo apt-get update
+3. sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
+
+Para abrir el programa, además de utilizar los comandos de navegación, una forma sencilla de ejecutar este tipo de archivos consiste en arrastrarlo hasta la ventana de la terminal, soltarlo ahí y luego presionar *entrar*.
+
+```
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: remove backticks around the jekyll figure code

**Comment by mariajoafana** [comment]
We need one more review and this will be ready to publish!

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+{% include figure.html filename="lfaligner-7.jpg" caption="Interfaz de selección de lenguas con la configuración deseada" %}
+
+**Cómo especificar las lenguas de los textos en las versiones de Linux y Mac**
+Como vimos anteriormente, las opciones se despliegan como texto dentro de la terminal. Debemos introducir con el teclado el número de lenguas (mínimo: 2; máximo: 11) y la combinación que deseamos, según el código de dos letras que cada lengua tiene (en nuestro caso: *es*, *en*, *fr* y *pt*).
+```
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: remove backticks around the jekyll figure code

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+c. Legislación de la UE según número CELEX.
+
+com. Propuestas de la Comisión Europea.
+
+epr. Informes del Parlamento Europeo.
+
+En este caso, ya que trabajaremos con documentos *txt*, debemos ingresar *t*- y presionar *entrar*.
+
+```
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: remove backticks around the jekyll figure code

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+Podemos observar que cada archivo está debidamente nombrado, con código de dos letras, según lengua. Al cargar todos los archivos, la interfaz se verá así:
+
+{% include figure.html filename="lfaligner-9.jpg" %}
+
+Presiona el botón *next* (siguiente) para que el software proceda con la alineación automática.
+
+**Carga de documentos en las versiones de Linux y Mac**
+Conforme lo solicite el programa, arrastraremos cada archivo -uno a la vez, en el orden que establecimos al momento de ingresar las lenguas-, lo soltaremos dentro de la ventana de la terminal y presionaremos *entrar*. Luego de haber cargado el último documento, el programa comenzará automáticamente con la alineación.
+```
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: remove backticks around the jekyll figure code

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+Luego de haber tomado una decisión al respecto -lo que también obedecerá a las necesidades de nuestro proyecto de investigación- debemos presionar el botón *next* (siguiente) para continuar.
+
+**Importante**. Seleccionar la opción de *Generate xls in background after review* (crear documento en formato xls después de la revisión), para poder exportar nuestro documento perfectamente alineado de manera automática, una vez completada la revisión.
+
+**Resultados de la alineación en las versiones de Linux y Mac**
+El programa nos informará sobre los resultados del proceso de alineación, ofreciéndonos casi las mismas opciones que su contraparte de Windows. La diferencia radica en que aquí simplemente nos pregunta si queremos revertir a segmentación por párrafo o no. Para tomar esta decisión, debemos basarnos en el resultado final de la segmentación que se muestra en pantalla:
+```
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: remove backticks around the jekyll figure code

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+## Ubicación del documento alineado
+Si seguiste las indicaciones anteriores sobre el nombramiento y almacenamiento de los textos, te será muy fácil encontrar el documento. Abre la carpeta en cuestión. Ahí verás una nueva carpeta cuyo nombre comienza con la palabra *align*. Dentro de ella encontrarás los documentos individuales en formato txt que corresponden al texto segmentado por el software y un archivo de planilla de datos (formato xls) que contiene el texto alineado y editado por nosotros.
+
+{% include figure.html filename="lfaligner-20.jpg" caption="El archivo exportado en formato xls aparecerá en la carpeta correspondiente" %}
+
+También puedes [**descargar**](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/align_2018.08.03_14.09.46/DDHH_es-es-en-fr-pt.xls) el documento alineado y explorarlo para aprender más sobre el funcionamiento de este software.
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: Please turn this into a relative link that points to `/assets/corpus-paralelo-lfaligner/...`

File: es/lecciones/corpus-paralelo-lfaligner.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+* Un texto de partida -digitalizado- y por lo menos una traducción de este. En este caso, alinearemos distintas traducciones de un documento que desde 1948 guía el quehacer y la convivencia humana en todos los ámbitos de la vida pública y privada, la [Declaración Universal de Derechos Humanos](https://es.wikipedia.org/wiki/Declaraci%C3%B3n_Universal_de_los_Derechos_Humanos): en [español](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_es.txt), [inglés](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_en.txt), [francés](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_fr.txt) y [portugués](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-paralelo-lfaligner/DDHH_pt.txt)
```
Comment (mdlincoln) [inline es/lecciones/corpus-paralelo-lfaligner.md]: Please turn these into relative links pointing to `/assets/corpus-paralelo-lfaligner...`

**Review by mdlincoln — CHANGES_REQUESTED** [review]

**Comment by mdlincoln** [comment]
Also, everyone please make sure to look at the deploy preview: https://deploy-preview-1242--ph-dev.netlify.com/es/lecciones/corpus-paralelo-lfaligner and be satisfied that everything is displaying correctly.

**Comment by arojascastro** [comment]
The abstract contains quotations marks that shouldn't be there and there are some images missing... i think

**Review by mariajoafana — COMMENTED** [review]
I just removed backticks and fixed relative links

**Review by mariajoafana — COMMENTED** [review]
fixed image link

**Review by mariajoafana — COMMENTED** [review]
fixed more broken links

**Comment by mariajoafana** [comment]
Image 3a link needs fixing, my connection is failing, but will try again in the afternoon

**Review by mariajoafana — COMMENTED** [review]
fixed link for image 3a

**Review by mariajoafana — COMMENTED** [review]
fixed link to image 4a

**Review by arojascastro — DISMISSED** [review]
cooooool

**Review by mdlincoln — APPROVED** [review]
publish whenever you're ready!

**Pull request merged**